modules/3_classification/src/classifier.cpp

```diff
@@ -11,11 +11,26 @@ using namespace cv::utils::fs;
 void topK(const std::vector<float>& src, unsigned k,
           std::vector<float>& dst,
           std::vector<unsigned>& indices) {
-    CV_Error(Error::StsNotImplemented, "topK");
+    std::vector<float> sorted;
+    sorted = src;
+    sort(sorted.begin(), sorted.end(), std::greater<float>());
+    for (int i = 0; i < k; ++i)
+    {
+        dst.push_back(sorted[i]);
+        auto foundEllement = std::find(src.begin(), src.end(), sorted[i]);
+        int indexEllement = std::distance(src.begin(), foundEllement);
+        indices.push_back(indexEllement);
+
+    }
 }
 
 void softmax(std::vector<float>& values) {
-    CV_Error(Error::StsNotImplemented, "softmax");
+    float sum = 0;
+    for (int i = 0; i < values.size(); ++i)
+        sum += exp(values[i]);
+    for (int i = 0; i < values.size(); ++i) {
+        values[i] = exp(values[i]) / sum;
+    }       
```

Comment:
Please failed test: https://travis-ci.com/github/dkurt/openvino_practice/builds/175185315

Reply:
> Please failed test: https://travis-ci.com/github/dkurt/openvino_practice/builds/175185315

Why? How do fix it?

Reply:
Try to modify a formula so it can work with large numbers but still produce valid results

Reply:
> Try to modify a formula so it can work with large numbers but still produce valid results

But what is sort of problem with large numbers?

Reply:
Try to print `exp(values[i])` to see

Reply:
> Try to print `exp(values[i])` to see

The Problem! How i can to return to branch classifier so that to do commit here?

Reply:
`git checkout classifier`

Reply:
> `git checkout classifier`

Tarvis write "job passed" but here "All checks have failed"

Reply:
this is the latest build: https://travis-ci.com/github/dkurt/openvino_practice/builds/175565357

You just need to restore origin `modules/4_detection/include/detector.hpp` file